packages/x-components/src/x-modules/history-queries/components/history-queries.vue

```diff
@@ -65,7 +66,7 @@
    * @public
    */
   @Component({
-    components: { BaseSuggestions, HistoryQuery },
+    components: { SearchInput, BaseSuggestions, HistoryQuery },
```

Comment:
Remove this

Reply:
Wow! You are right, i think (hope) that IDE added this. I am removing it.